.gitignore

```diff
@@ -28,10 +28,10 @@ config.rst
 
 package-lock.json
 
-share/jupyter/voila/templates/nbconvert/static/index.css
-share/jupyter/voila/templates/nbconvert/static/theme-light.css
-share/jupyter/voila/templates/nbconvert/static/theme-dark.css
-share/jupyter/voila/templates/default/static/*voila.js
-share/jupyter/voila/templates/default/static/*[woff|woff2|eot|svg]
+share/jupyter/templates/voila/default/resources/index.css
+share/jupyter/templates/voila/default/resources/theme-light.css
+share/jupyter/templates/voila/default/resources/theme-dark.css
+share/jupyter/templates/voila/default/resources/*voila.js
```

Comment:
What is the reason for renaming `static` to `resources`?

Reply:
The directory is used by both nbconvert and voila, voila would serve it under static*, nbconvert will include the files. Naming it static would feel wrong for nbconvert, but naming it resources wouldn't feel wrong for voila. What do you think?

*:   maybe we should rename that as well, to make life/voila simpler. E.g http://localhost:8866/voila/resources/index.css

Reply:
It think it makes sense :)